uk_bin_collection/uk_bin_collection/councils/LondonBoroughRedbridge.py

```
@@ -94,27 +94,14 @@ def extract_collection_data(collection_div, collection_type):
-                            # Check if the month is ahead of the current month
-                            if collection_date_obj.month >= current_month:
-                                # If the month is ahead, use the current year
-                                formatted_date = collection_date_obj.strftime(
-                                    date_format
-                                )
-                            else:
-                                # If the month is before the current month, use the next year
-                                formatted_date = collection_date_obj.replace(
-                                    year=current_year + 1
-                                ).strftime(date_format)
+                            ).strftime(date_format)
```

Ruff (0.14.14)

[error] `date_format` may be undefined, or defined from star imports

(F405)